modin/pandas/test/test_io.py

```diff
@@ -1836,6 +1845,10 @@ def test_to_stata(self):
 
 
 class TestFeather:
+    @pytest.mark.skipif(
```

Comment:
```suggestion
    @pytest.mark.xfail(
```

Reply:
It seems `xfail` couldn't take a condition in former times. Applied.

Reply:
Yes, it's a good addition. We should change all of our `skipif` to `xfail` in the test suite.

Reply:
That is what you want to resolve as a part of #2846 ?

Reply:
We can do it there, yes.